### DISCUSSION TITLE
How to supply `FIREBASE_APP_CHECK_DEBUG_TOKEN` to AppCheck when building with android studio

### DISCUSSION THREAD

**Discussion by rawatnaresh** [invertase/react-native-firebase#6770] [Q&A]
we use Android studio to build our react-native app and I'm not sure if its possible to pass `FIREBASE_APP_CHECK_DEBUG_TOKEN` using android studio.

I believe most of us use some package to manage our environment variables. In our case we are using [react-native-config](https://github.com/luggit/react-native-config). It would be so much easier if we could just define `FIREBASE_APP_CHECK_DEBUG_TOKEN` in our `.env` file and everything work out as expected. This way we don't have to update env vars at multiple places.

Also, I'm wondering why we haven't use this approach to pass debug token instead of defining as a system variable

```js
firebase.appCheck().activate('ignored', false, false, 'my_debug_token');
```

**Comment by mikehardy** [comment]
I'm open to reasonable PRs, if you are motivated to contribute different ways to inject the debug token, but the API you propose won't work as it does not conform to the firebase-js-sdk APIs, and they are our model, we use them as our guide.
